juel/src/main/java/org/operaton/bpm/impl/juel/NumberOperations.java

```diff
@@ -32,9 +32,7 @@ private final static boolean isDotEe(String value) {
 		int length = value.length();
 		for (int i = 0; i < length; i++) {
 			switch (value.charAt(i)) {
-				case '.':
-				case 'E':
-				case 'e': return true;
+				case '.', 'E', 'e': return true;
```

Comment:
Maybe refactor this switch with a set#contains check? Having a switch with one return statement reads a little funny to me